lib/util/sdp/simulcast.js

```diff
@@ -179,6 +185,18 @@ function createTrackIdsToAttributes(section) {
   const ssrcAttrTuples = getMatches(section, '^a=ssrc:([0-9]+) msid:([^\\s]+) ([^\\s]+)$');
   const rtxPairs = getSSRCRtxPairs(section);
 
+  // If no line of the form "a=ssrc:123 msid:foo bar" exists, then we are most
+  // probably dealing with a "unified-plan" m= section.
+  if (ssrcAttrTuples.length === 0) {
+    const trackId = flatMap(getMatches(section, '^a=msid:.+ (.+)$'))[0];
+    const ssrcs = flatMap(getMatches(section, '^a=ssrc:(.+) cname:.+$'));
+    if (trackId) {
+      ssrcs.map(ssrc => [ssrc, null, trackId]).forEach(tuple => {
+        ssrcAttrTuples.push(tuple);
+      });
+    }
+  }
```

Comment:
Can we structure this code differently? When we call this code, we know what SDP format we are using. So can we not parameterize this function by the SDP format rather than use `ssrcAttrTuples.length === 0`?`ssrcAttrTuples.length === 0` seems indirect and might not actually indicate Unified Plan.